homeassistant/components/weather/ipma.py

```diff
@@ -0,0 +1,177 @@
+"""
+Support for IPMA weather service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/weather.ipma/
+"""
+import logging
+from datetime import timedelta
+import async_timeout
```

Comment:
Group 3rd party imports and have a blank line between the three groups standard library, 3rd party and homeassistant imports.